libsignal-service/src/utils.rs

```diff
@@ -1,4 +1,18 @@
+// Signal sometimes adds padding, sometimes it does not.
+// This requires a custom decoding engine.
+// This engine is as general as possible.
+pub const BASE64: base64::engine::GeneralPurpose =
```

Comment:
I would give this a slightly more descriptive name, like this:

```suggestion
pub const BASE64_RELAXED: base64::engine::GeneralPurpose =
```

If you want, I'll run Rust analyzer over the source and get that committed, if you don't have a rename command around.

Other name suggestions are welcome too.

Reply:
Renamed as desired.